packages/react-styles/src/css/components/Topology/topology-pipelines.css

```diff
@@ -0,0 +1,362 @@
+  --pf-topology-pipelines__pill-status--color: var(--pf-topology-pipelines__pill-status--m-selected--m-warning--color);
+}
+.pf-topology-pipelines__pill-status.pf-m-selected.pf-m-in-progress {
+  --pf-topology-pipelines__pill-status--color: var(--pf-topology-pipelines__pill-status--m-selected--m-in-progress--color);
+}
+.pf-topology-pipelines__pill-status.pf-m-selected.pf-m-pending {
+  --pf-topology-pipelines__pill-status--color: var(--pf-topology-pipelines__pill-status--m-selected--m-pending--color);
+}
+.pf-topology-pipelines__pill-status.pf-m-selected.pf-m-running {
+  --pf-topology-pipelines__pill-status--color: var(--pf-topology-pipelines__pill-status--m-selected--m-running--color);
+}
+.pf-topology-pipelines__pill-status.pf-m-selected.pf-m-idle {
+  --pf-topology-pipelines__pill-status--color: var(--pf-topology-pipelines__pill-status--m-selected--m-idle--color);
+}
+.pf-topology-pipelines__pill-status.pf-m-spin {
+  filter: blur(0);
+  -webkit-filter: blur(0);
+  transform-box: fill-box;
+  transform-origin: center;
+  animation: status-spin 2s infinite linear;
+}
+@keyframes status-spin {
+  0% {
+    -webkit-transform: rotate(0deg);
+    transform: rotate(0deg);
+  }
+  100% {
+    -webkit-transform: rotate(359deg);
+    transform: rotate(359deg);
+  }}
+
+.pf-topology-pipelines__pill.pf-m-selectable {
+  cursor: pointer;
+}
+.pf-topology-pipelines__pill.pf-m-selected .pf-topology__node__action-icon__icon {
+  color: var(--pf-topology-pipelines__pill-text--m-selected--Color);
+}
+.pf-topology-pipelines__pill.pf-m-selected.pf-m-danger .pf-topology__node__action-icon__icon {
+  color: var(--pf-topology-pipelines__pill-text--m-selected--m-danger--Color);
+}
+.pf-topology-pipelines__pill.pf-m-selected.pf-m-success .pf-topology__node__action-icon__icon {
+  color: var(--pf-topology-pipelines__pill-text--m-selected--m-success--Color);
+}
+.pf-topology-pipelines__pill.pf-m-selected.pf-m-warning .pf-topology__node__action-icon__icon {
+  color: var(--pf-topology-pipelines__pill-text--m-selected--m-warning--Color);
+}
+.pf-topology-pipelines__pill.pf-m-selected.pf-m-in-progress .pf-topology__node__action-icon__icon {
+  color: var(--pf-topology-pipelines__pill-text--m-selected--m-in-progress--Color);
+}
+.pf-topology-pipelines__pill.pf-m-selected.pf-m-pending .pf-topology__node__action-icon__icon {
+  color: var(--pf-topology-pipelines__pill-text--m-selected--m-pending--Color);
+}
+.pf-topology-pipelines__pill.pf-m-selected.pf-m-running .pf-topology__node__action-icon__icon {
+  color: var(--pf-topology-pipelines__pill-text--m-selected--m-running--Color);
+}
+.pf-topology-pipelines__pill.pf-m-selected.pf-m-idle .pf-topology__node__action-icon__icon {
+  color: var(--pf-topology-pipelines__pill-text--m-selected--m-idle--Color);
+}
+
+.pf-topology-pipelines__when-expression--background {
+  stroke: var(--pf-topology-pipelines__when-expression--background--Stroke);
+  fill: var(--pf-topology-pipelines__when-expression--background--Fill);
```

Comment:
I think you should be able to reassign this variable in the modifiers below (lines 344-358)
I'd probably rename the class and variables with `when-expression--background` and `--edge` to just use one `-` like `when-expression-background` and `when-expression-edge` since it's an element, not an attribute.